platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/internal/FakeViewConnection.kt

```diff
@@ -50,6 +50,8 @@ internal class FakeViewActionCollector(
             ViewAction.ToggleQuote -> toggleQuote()
             ViewAction.Undo -> undo()
             ViewAction.Unindent -> unindent()
+            is ViewAction.ReplaceSuggestionText -> Unit
+            is ViewAction.InsertMentionAtSuggestion -> Unit
```

Comment:
Just FYI, leaving these no-ops will make it impossible to test this behaviour in host apps.

Reply:
I'm wondering what I should do here, I can replace the `state.menuAction` with `MenuAction.None`, thus 'removing' the suggestion? It's not like we can or should modify the underlying text here, right?

Reply:
The idea isn't that it works correctly, only that something in the state changes which is visible to the test. If you wanted to implement it, you could do something simple like search and replace the final occurrence of `@.*`.

Reply:
Ok, I'll do just that 👍 .